src/dappsExplorer/DappCard.tsx

```diff
@@ -98,22 +91,28 @@ function DappCard({
   onFavoriteDapp,
   disableFavoriting,
   showBorder,
+  cardContentContainerStyle,
+  cardStyle,
   testID,
 }: Props) {
   return (
     <Card
       testID={testID}
-      style={[styles.card, showBorder ? styles.borderStyle : {}]}
+      style={[styles.card, showBorder ? styles.borderStyle : {}, cardStyle]}
       rounded={true}
       shadow={null}
     >
-      <Touchable onPress={onPressDapp} borderRadius={8} testID={`Dapp/${dapp.id}`}>
+      <Touchable
+        style={[styles.pressableCard, cardContentContainerStyle]}
+        onPress={onPressDapp}
+        borderRadius={showBorder ? 8 : undefined}
```

Comment:
nit: can 8 be `Spacing.Smallest8`?

Reply:
i prefer not to 🙈 i agree that we prefer variables over hardcoded numbers so that we are restricted to certain consistent values + it's easier to make a global change for the values, but i think borders and spaces may not want to use the same variables...it can be a surprise if a change to "Spacing" values also changes rounded corners. i've always wanted to create a separate style enum for borders but never got round to it 😅